Bottom: 6ddb54f27f5d5e161da54366daf63c9cbba0ed9b
Top:    f6ccd17e23ac399cd4c50553197c40209aeee0fa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 13:31:43 +0100

Refresh of add-support-for-a-poured-objec-I

---

diff --git a/src/polygon.c b/src/polygon.c
index 66c2706..45a2103 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -729,7 +729,6 @@ InitClip (DataTypePtr Data, LayerTypePtr layer, PolygonType * p)
     p->NoHolesValid = 0;
   return 1;
 #endif
->>>>>>> patched
 }
 
 /* find polygon holes in range, then call the callback function for
